pages/common/jhsdb.md

```diff
@@ -0,0 +1,20 @@
+# jhsdb
+
+> Attach to a Java process or launch a postmortem debugger to analyze the core dump from a crashed Java Virtual Machine.
+> More information: <https://manned.org/jhsdb>.
+
+- Use a PID to connect to a hung process:
+
+`jhsdb jstack --pid {{pid}}`
+
+- Open a core dump for analysis:
```

Comment:
What do you mean by `open`?

Reply:
Is `Open a core dump in interactive debug mode:` better?

Reply:
Yes, the `interactive` is important here. Otherwise it sounds like it is opened and then the command finishes.